BurnOutSharp/ProtectionType/AACS.cs

```diff
@@ -12,23 +12,57 @@ public string CheckPath(string path, bool isDirectory, IEnumerable<string> files
         {
             if (isDirectory)
             {
-                if (files.Any(f => f.Contains(Path.Combine("aacs", "VTKF000.AACS")))
-                    && files.Any(f => f.Contains(Path.Combine("AACS", "CPSUnit00001.cci"))))
+                if (files.Any(f => f.Contains(Path.Combine("AACS", "MKBROM.AACS"))))
                 {
-                    return "AACS";
+                    string file = files.FirstOrDefault(f => Path.GetFileName(f).Equals("MKBROM.AACS", StringComparison.OrdinalIgnoreCase));
+                    int? version = GetVersion(file);
+                    return (version == null ? "AACS (Unknown Version)" : $"AACS Version {version}");
+                }
+                if (files.Any(f => f.Contains(Path.Combine("AACS", "MKB_RO.inf"))))
+                {
+                    string file = files.FirstOrDefault(f => Path.GetFileName(f).Equals("MKB_RO.inf", StringComparison.OrdinalIgnoreCase));
+                    int? version = GetVersion(file);
+                    return (version == null ? "AACS (Unknown Version)" : $"AACS Version {version}");
                 }
             }
             else
             {
                 string filename = Path.GetFileName(path);
-                if (filename.Equals("VTKF000.AACS", StringComparison.OrdinalIgnoreCase)
-                    || filename.Equals("CPSUnit00001.cci", StringComparison.OrdinalIgnoreCase))
+                if (filename.Equals("MKBROM.AACS", StringComparison.OrdinalIgnoreCase))
                 {
-                    return "AACS";
+                    string file = path;
+                    int? version = GetVersion(file);
+                    return (version == null ? "AACS (Unknown Version)" : $"AACS Version {version}");
                 }
+                if (filename.Equals("MKB_RO.inf", StringComparison.OrdinalIgnoreCase))
+                {
+                    string file = path;
+                    int? version = GetVersion(file);
+                    return (version == null ? "AACS (Unknown Version)" : $"AACS Version {version}");
+                } 
             }
 
             return null;
         }
+        private static int? GetVersion(string path)
+        {
+            if (File.Exists(path))
```

Comment:
Invert this `if` please. So say "if the file doesn't exist, return null". It's a bit cleaner that way. It reduces an unnecessary level of nesting.